Makefile

```
@@ -1,16 +1,18 @@
 all: test
 
 init:
-	-dropdb spgq
-	createdb spgq
+	dropdb --if-exists spgq
+	-createdb spgq
```

So database creation error is ignored? Why?